### DISCUSSION TITLE
supabaseUrl is required on Netlify prod Nuxt 3

### DISCUSSION THREAD

**Discussion by JhinAhri12** [nuxt/nuxt#19300] [Questions] [answered]
Hi

I have try everything that i know for deploy a website with supabase and nuxt but it say error 500 supabaseUrl is required.

I have a dotenv and i have try to set many times env variable on netlify but it dont work.

Someone have facing this problem ?

Thanks for help.

![](https://user-images.githubusercontent.com/31922998/221429193-5c1ca234-142e-41e2-bf21-4a367aba9d48.png)

![221429252-7ab11c87-1e61-4ef7-b3f5-f87410062478](https://user-images.githubusercontent.com/31922998/221508518-ce86e73f-2f33-41d4-a1f4-edf9f6e001f6.png)

**Comment by AloisSeckar** [comment, marked as answer]
My Netlify deploys seem to work just fine. I go **"Site Settings"** and then **"Environmental variables"**, set the values here:

![image](https://user-images.githubusercontent.com/7399922/221512506-641e3971-32d2-425d-b62e-742c0daff927.png)

Don't think there should be anything else to do. My build command is: **npm ci && npm run build**

**Reply by Baroshem** [reply]
I agree with @AloisSeckar 

There shouldn't be anything else to do. You can also take a look at my article and video where I am building a simple Nuxt app with Supabase and deploy to Vercel

https://dev.to/jacobandrewsky/nuxt-3-tailwind-and-supabase-crash-course-1482
https://www.youtube.com/watch?v=A-G6lPHctPs

**Comment by JhinAhri12** [comment]
Hi thanks for your response. I have reset environment variables and the error 500 is still here :/
![image](https://user-images.githubusercontent.com/31922998/221523156-c0001efd-27a4-426f-b7ed-5fd37b0d5587.png)
![image](https://user-images.githubusercontent.com/31922998/221523638-850b4dad-2f23-4991-bd3a-009bf07c64e9.png)

I go try deploy with vercel and your tutorial. Thanks for the videos ! :)

**Reply by JhinAhri12** [reply]
It work on Vercel so I think its a bug on netlify config ? But no problem the website is deploy that's ok.
Thanks for your help !
